backend/models/organisation/model.py

```diff
@@ -1,28 +1,75 @@
-from odmantic import Field, Model
+from odmantic import Field, Model, EmbeddedModel
 from pydantic import EmailStr
 from typing import Optional
 
+class OrganisationData(EmbeddedModel):
+    location: str
+    companyBio: str
+    industry: str
+    telephone: str
+    companySize: Optional[str] = None
+    
+    class Config:
+        scheme_extra = {
+            "example": {
+                "location": "USA",
+                "companyBio": "Google is the largest technology company",
+                "industry": "Technology",
+                "telephone": "+447398346125",
+                "companySize": "10,000+ Employees"
+            }
+        }
+
 
 class Organisation(Model):
-    id: int
     firstName: str
     lastName: str
     email: EmailStr
     password: str
     companyName: str
-    location: str
-    companyBio: str
-    industry: str
-    telephone: str
-    companySize: Optional[str] = None
+    data: Optional[OrganisationData]
+        
+    class Config:
+        scheme_extra = {
+            "example": {
+                "firstName": "Larry",
+                "lastName": "Page",
+                "email": "page@gmail.com",
+                "password": "Strong+Password",
+                "companyName": "Google"
+            }
+        }
+        
     
 class Job(Model):
-    id: int
     title: str
     role: str
     description: str
     industry: str
     company: Organisation
     experience: str
     salary: float
-    
\ No newline at end of file
+        
+    class Config:
+        schema_extra = {
+            "example": {
+                "title": "Software Engineer",
+                "role": "Junior",
+                "description": "Python developer needed to develop the Backend",
+                "industry": "Internet and Technology",
+                "company": { 
+                    "firstName": "Larry",
+                    "lastName": "Page",
+                    "email": "page@gmail.com",
+                    "password": "Strong+Password",
+                    "companyName": "Google",
+                    "location": "USA",
+                    "companyBio": "Google is the largest technology company",
+                    "industry": "Technology",
+                    "telephone": "+447398346125",
+                    "companySize": "10,000+ Employees"
```

Comment:
It will be nice to limit the data the user can see about the organisation so I'll fix this.